include/LLGL/StreamOutputAttribute.h

```diff
@@ -11,6 +11,7 @@
 
 #include "Export.h"
 #include "Format.h"
+
```

Comment:
Generally not a problem for me when you do your formating here as you like, but in order to keep this pull request simpler, please revert all these blank lines between include headers you added.

Reply:
They are intentional in this particular case, because i want to see what headers are included. Might change all includes in the future.